log/log_ext.go

```diff
@@ -1,6 +1,6 @@
 // (c) 2019-2025, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
-package customtypes
+package log
```

Comment:
If we do move this here, let's call it `customlog` since we import it as `customlog`